src/app/shared/search/import { CaseField } from '..:domain:definition:ca

```diff
@@ -0,0 +1,8 @@
+import { CaseField } from '../domain/definition/case-field.model';
+
+export class SearchResultViewItem {
+  case_id: string;
+  case_fields: object;
+  hydrated_case_fields?: CaseField[];
+  columns?: object;
```

Comment:
can we have `columns` declared with a type? We could use ES6 Map

Reply:
👍